tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -746,8 +798,9 @@ InvokeRule(r, argExprsChunks) ::= <<
 	var _x = p.<r.name; format="cap">(<argExprsChunks>)
 	<endif>
 
+    <r.labels:{l | <labelRefStuff(l,"_x")> }; separator="\n">
```

Comment:
it is only styling. go fmt was fix it. it only matter if there are style checkers as part of commit code to a repo and you commit generated code.